ocp_install_env.sh

```diff
@@ -107,9 +107,14 @@ controlPlane:
     baremetal: {}
 platform:
   baremetal:
+    libvirtURI: qemu+ssh://${PROVISIONING_HOST_USER}@${PROVISIONING_HOST_IP}/system
```

Comment:
Was this intentional to include? Shouldn't it be optional since the default is `qemu:///system`?

Reply:
I assume that's why the latest dev-scripts run failed

Reply:
I need it there for hive, and I was trying to just be consistent and always include it. I'll make it configurable and deal with that in a separate PR.